packages/manifest-validator/src/schema/schema.ts

```diff
@@ -68,8 +72,16 @@ export const schema = {
       format: "uri",
     },
     icon: {
-      type: "string",
-      format: "uri",
+      oneOf: [
+        {
+          type: "string",
+          format: "uri",
+        },
+        {
+          type: "string",
+          maxLength: 0,
+        },
```

Comment:
Is it to make the field icon optional or support `""` ?
We probably should remove the field instead if a manifest have an icon as an empty string, wdyt ?